services/net/indexing/IndexingManager.cs

```diff
@@ -286,10 +286,15 @@ private async Task IndexContentAsync(ContentModel content)
     /// <returns></returns>
     private async Task PublishContentAsync(ContentModel content)
     {
+        // Remove the transcript body if it hasn't been approved.
```

Comment:
I don't love this implementation, but it'll work for our needs presently.  It is also very easy to implement.